terraform/crates-io-ops-bot/deployment.tf

```diff
@@ -0,0 +1,105 @@
+// Deploys the bot to the production ECS cluster
+
+// The ECR repository is used by CI to store the container image, which will
+// then be fetched by the ECS service.
+module "ecr" {
+  source = "../shared/modules/ecr-repo"
+  name   = "crates-io-ops-bot"
+}
+
+// This just loads the data for each of these secrets
+// The values are added to ASM manually
+data "aws_ssm_parameter" "crates_io_ops_bot" {
+  for_each = toset([
+    "discord_token",
+    "heroku_api_key",
+    "build_check_interval",
+    "github_org",
+    "github_repo",
+    "github_token"
+  ])
+  name = "/prod/ecs/crates_io_ops_bot/${each.value}"
+}
+
+module "ecs_task" {
+  source = "../shared/modules/ecs-task"
+
+  name               = "crates_io_ops"
+  cpu                = 256
+  memory             = 512
+  log_retention_days = 7
+  ecr_repositories_arns = [
+    module.ecr.arn,
+  ]
+
+  containers = <<EOF
+[
+  {
+    "name": "app",
+    "image": "${module.ecr.url}",
+    "essential": true,
+    "portMappings": [
+      {
+        "containerPort": 80,
+        "hostPort": 80
+      }
+    ],
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "/ecscratesioops",
+        "awslogs-region": "us-west-1",
+        "awslogs-stream-prefix": "cratesioops"
```

Comment:
Let's call this `crates-io-ops-bot` to be consistent with the rest of the tasks.